src/cli/commands/wallet.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @module cli/commands
+ */
+
+import {CliCommand} from "./interface";
+import {CommanderStatic} from "commander";
+import logger from "../../logger";
+import fs from "fs";
+import {CliError} from "../error";
+import bls from '@chainsafe/bls-js';
+import { blsPrivateKeyToHex } from '../../util/bytes';
+
+interface IWalletCommandOptions {
+  outputFile: string;
+}
+
+export class CreateWalletCommand implements CliCommand {
+  public register(commander: CommanderStatic): void {
+    commander
+      .command("wallet")
+      .description("Generate wallet private key")
+      .option("-o, --outputFile [output_file]", "Path to output file destination", "keys/validator/bls.json")
+      .action(async (options) => {
+        // library is not awaiting this method so don't allow error propagation 
+        // (unhandled promise rejections)
+        try {
+          await this.action(options);
+        } catch (e) {
+          logger.error(e.message);
+        }
+      });
+  }
+
+  public async action(options: IWalletCommandOptions): Promise<void> {
+    if (fs.existsSync(options.outputFile)) {
+      throw new CliError(`${options.outputFile} already exists`);
+    }
+
+    const keyPair = bls.generateKeyPair();
+
+    const object = {
+      privateKey: blsPrivateKeyToHex(keyPair.privateKey),
+      // publicKey: keyPair.publicKey.toHexString()
+    }
+
+    try {
+      fs.writeFileSync(options.outputFile, JSON.stringify(object, null, 2));
```

Comment:
You would wan't to ask user for a password and encrypt the data with symetric encryiption. I would suggest to call class doing that (encrypt and save file) `Keystore` because later on it will be modified to use agreeed keystore format. This class should also have method to load private key file, decrypt and return keypair

Reply:
How ethers handles this:
Pre-Checks: 
https://github.com/ethers-io/ethers.js/blob/master/wallet.js#L119
Actual encryption process:
https://github.com/ethers-io/ethers.js/blob/master/utils/secret-storage.js#L233

@mpetrunic We could do a very simple version for the time being, or leave it for another PR

Reply:
okay yeah I don't usually work with security related stuff so excuse me if this question is bad but what is the point of symmetric encryption when all of the code and the secret would be open-sourced? How does that add security for the key?

In any case, should I still abstract the logic to this `Keystore` class (and including the actual saving of the key)?

Reply:
Symmetric encryption allows the same key to encrypt and decrypt.

The idea is that the user knows the password, but no one else does.

Reply:
Would you like me to do this now or save for another PR?

Reply:
> Symmetric encryption allows the same key to encrypt and decrypt.
> 
> The idea is that the user knows the password, but no one else does.

Yeah, I hadn't read the part about entering a pw for encrypting the key and assumed it was through a secret. I totally get the need for it.

Reply:
I think it would be wise to do simple version with just symmetric encryption in this PR

Reply:
@mpetrunic There is no current logic or implementation for using/decrypting the private key, should I just create the functions to test anyway?

Reply:
Yeah implement them, it will be easier to test